ngageoint/opensphere
# WIP: feat(olcs): Use OL tile load functions rather than Cesium's

This is related to (and based on) openlayers/ol-cesium#562 and openlayers/ol-cesium#37. However, we need it for GeoPackage support rather than for raster reprojection. Additionally, this helps support any other custom tile classes and tile load functions in Cesium.

Testing:
-  Check out base maps in EPSG:3857 and EPSG:4326 and ensure they load fine in Chrome, Firefox, and IE (I did have an issue with Firefox on a specific machine, but I'll have to get back on there and see what version it was)
- Check out raster data layers (timed layers from GeoServer or ArcGIS) and ensure they work. Apply a custom color to them and ensure that works as well.
- For timed raster layers, open the timeline and ensure that the layer continues to animate properly in playback
